templates/tree-editor/example-file/example-file-command.ts

```diff
@@ -23,8 +24,14 @@ export class NewTreeExampleFileCommandHandler implements SingleUriCommandHandler
         protected readonly fileService: FileService,
         @inject(ILogger)
         protected readonly logger: ILogger,
+        @inject(WorkspaceService)
+        protected readonly workspaceService: WorkspaceService
     ) { }
     
+    isEnabled(uri: URI){
```

Comment:
We do not make use of the passed `uri`. Is this for extension developers to update, or should we remove it?